### PR TITLE
Add missing responses

### PR DESCRIPTION
**Pull request by BrainMaestro**
Part of #62

### CODE REVIEW

**Comment by BrainMaestro** [comment]
Approved

**Comment by SkaeX** [comment]
Approved.

File: tests/unit/Controllers/SectionControllerTest.php [hunk 1]
```diff
@@ -165,8 +165,10 @@ public function testUpdate()
             ->method('updateSection')
             ->with($id, $request->all())->willReturn(true);
```
Comment (tenorz007) [inline tests/unit/Controllers/SectionControllerTest.php]: why no parent::ID? @BrainMaestro

Comment (BrainMaestro) [inline tests/unit/Controllers/SectionControllerTest.php]: `$id` was set to `parent::ID` initially. This file is different from the others. There should be more consistency though

**Pull request merged**